cygnus-twitter/src/test/java/com/telefonica/iot/cygnus/sinks/TwitterHDFSSinkTest.java

```diff
@@ -0,0 +1,103 @@
+package com.telefonica.iot.cygnus.sinks;
+
+import com.telefonica.iot.cygnus.backends.hdfs.HDFSBackendImplREST;
+import org.apache.flume.Context;
+import org.apache.flume.channel.MemoryChannel;
+import org.apache.flume.lifecycle.LifecycleState;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author jpalanca
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class TwitterHDFSSinkTest {
+
+    // mocks
+    @Mock
+    private HDFSBackendImplREST mockWebHDFSBackend;
+
+    // instance to be tested
+    private TwitterHDFSSink sink;
+
+    // context constants
+    private final String[] cosmosHost = {"localhost"};
+    private final String cosmosPort = "14000";
+    private final String hdfsUsername = "user1";
+    private final String hdfsPassword = "12345";
+    private final String hdfsFolder = "test_folder";
+    private final String hdfsFile = "test_file.txt";
+    private final String oauth2Token = "tokenabcdefghijk";
+    private final String enableHive = "false";
+    private final String enableKrb5Auth = "false";
+
+    @Before
+    public void setUp() throws Exception {
+        // set up the instance of the tested class
+        sink = new TwitterHDFSSink();
+        sink.setPersistenceBackend(mockWebHDFSBackend);
+
+        // set up the behaviour of the mocked classes
+        when(mockWebHDFSBackend.exists(null)).thenReturn(true);
+        doNothing().doThrow(new Exception()).when(mockWebHDFSBackend).createDir(null);
+        doNothing().doThrow(new Exception()).when(mockWebHDFSBackend).createFile(null, null);
+        doNothing().doThrow(new Exception()).when(mockWebHDFSBackend).append(null, null);
+    } // setUp
+
+
+    /**
+     * Test of configure method, of class TwitterHDFSSink.
+     */
+    @Test
+    public void testConfigure() {
```

Comment:
Please, have a look on how latests tests added to Cygnus have been created, e.g. https://github.com/telefonicaid/fiware-cygnus/blob/develop/cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSICartoDBSinkTest.java As you can see, there is a utility method named `getTestTraceHead` that helps us to align the test traces. Please observe all the tests must produce a `-  OK  -  Bla bla bla` trace or a `- FAIL - Bla bla bla` trace.